libs/blocks/action-item/action-item.css

```diff
@@ -133,8 +133,12 @@
   transform: scale(1.05);
 }
 
+.action-item.float-icon .main-image {
+  display: block;
+}
+
```

Comment:
Is there a reason why you don't want `.action-item .main-image` to be a block level element? 

I noticed `.main-image` is added to elements that are inline by default (`<strong>` and `<p>`). If you declare those to be block level it doesn't look like you need to change `.action-item.float-icon .floated-icon` to be `inline-flex`. One less change, and one less thing altered. :). Maybe i'm missing something?

Reply:
@rgclayton You make a solid point. I was really just trying to avoid any conflicts with the other variants by only making this change for the float icon variant.

Reply:
@rgclayton okay I checked it out and reverted my `inline-flex`. Thanks for looking out!